### ISSUE TITLE
Ink Story converts floats based on user system regional settings

### ISSUE THREAD

**Issue by tarnos12** [inkle/ink-unity-integration#207]
I have a problem with Unity reading `0.234` as `0,234` when I parse a tag

This can be fixed by changing system regional format to USA(so it uses a dot instead of a comma)

Problem is that this does not save when the game is compiled, so other users will experience issues if their regional format is not set to something that uses dot for a float.

Is there something that I might need to change in Unity or ink compiler/config?

`"audioambient:asset\'ambient_sanctuary_deep\',volume\'0,15\',loop\'0\'"`

This is the tag from `story.CurrentTags` where volume is `0,15` instead of `0.15` which is based on my regional settings.

**Comment by tomkail** [comment]
Hi! This definitely sounds like a problem. Can you move this ticket to the
ink discord? This is for the Unity plugin.

**Comment by tarnos12** [comment]
Hey, I thought it might be related to Unity plugin.
The issue happens after the game is compiled, so the plugin is the one reading the json and converting it into readable format?

`story.CurrentTags` gives me values with a comma(or a dot based on my system settings)

**Comment by tomkail** [comment]
Ah I see. The plugin makes use of the runtime and the compiler, but isn’t
where that code is developed. This plugin is more about the auto
compilation and unity editor tools.

**Comment by tarnos12** [comment]
Sorry for the confusion.
Do you know specific channel to ask this question on the discord?
I found Unity integration, but that's the same as this.
Inky-engine-dev perhaps?

**Comment by tomkail** [comment]
Yes that would be right! Or just the ink channel. Tag Joe humphfey in it,
he’s the lead dev

**Comment by tarnos12** [comment]
Thanks, I have done what you said.
We have figured out that the issue happens with inky variables converting floats without Invariant Culture.
Probably with `.toString()`

This issue can be closed.
